packages/amplify-graphql-auth-transformer/src/utils/index.ts

```diff
@@ -27,6 +33,71 @@ export const splitRoles = (roles: Array<RoleDefinition>): RolesByProvider => ({
   lambdaRoles: roles.filter(r => r.provider === 'function'),
 });
 
+/**
+ * returns @auth directive rules
+ */
+export const getAuthDirectiveRules = (authDir: DirectiveWrapper): AuthRule[] => {
+  const splitReadOperation = (rule: AuthRule): void => {
+    const operations: (ModelOperation | 'read')[] = rule.operations ?? [];
+    const indexOfRead = operations.indexOf('read', 0);
+    if (indexOfRead !== -1) {
+      operations.splice(indexOfRead, 1);
+      operations.push('get');
```

Comment:
I see that `read` is being expanded to `get, list, search, sync, listen`. Please confirm this with others if grouping all these ops together under `read` is the right approach.

Reply:
This is today's behavior, changing this will end up changing the behavior on the existing customers applications. 
We did have a design review on this change.

Reply:
I am fine with this if this went through a review.